cloud-init.sh

```diff
@@ -283,7 +283,7 @@ if [ "$EE_LICENSE" != "placeholder" ]; then
     cat <<EOF >> /etc/kong/kong.conf
 enforce_rbac = on
 admin_gui_auth = basic-auth
-admin_gui_session_conf = { "secret":"${SESSION_SECRET}", "cookie_secure":false }
```

Comment:
> "cookie_samesite":"<SET_DEPENDING_ON_DOMAIN>": "Strict" by default. See [Session Security](https://docs.konghq.com/gateway/latest/configure/auth/kong-manager/sessions/#session-security) for exceptions.

Note that `cookie_samesite` defaults to `Strict` so `None` might not be suitable as a default option. How about change it into a tf variable?